.storybook/themes/news.scss

```diff
@@ -1828,6 +1875,13 @@
 				article-body-gallery-additional-previous: (
 					display: block,
 				),
+				author-bio-author-name: (
+					font-size: var(--heading-level-5-font-size),
```

Comment:
This should be `heading-level-6` per above, so if you set that, this can be removed.

Reply:
Removed this, because author-name font size is same in both desktop and default